params: output changed parameter timestamps in csv

tstastny opened the pull request:
Outputs changed parameters and their timestamps in csv. This is just one way to format this -- open to other ideas. "0" is used as the timestamp for the initial parameter.

resolves #26 

Output in the csv formatted like this:

| column 1      | column 2      | column 3      | column 4      |
| ------------- | ------------- | ------------- | ------------- |
| PARAM 1       | init val      | changed val 1 | changed val 2 |
| timestamp     | 0             | timestamp 1   | timestamp 2   |
| PARAM 2       | init val      | changed val 1 | changed val 2 |
| timestamp     | 0             | timestamp 1   | timestamp 2   |

Ex. sample output from a ulog:

| column 1         | column 2         | column 3         | column 4         |
| ---------------- | ---------------- | ---------------- | ---------------- |
| FW_RLL_TO_YAW_FF | 0                |                  |                  |
| timestamp        | 0                |                  |                  |
| FW_RR_FF         | 0.8999999762     | 1.2000000477     | 1                |
| timestamp        | 0                | 429617575        | 791731803        |
| FW_RR_I          | 0.1000000015     |                  |                  |
| timestamp        | 0                |                  |                  |

bkueng commented:
Thanks @tstastny . The format is fine with me, but can we change the default to not print the timestamps? E.g. by adding a flag `-t/--timestamps`.

BazookaJoe1900 commented:
looks good

BazookaJoe1900 commented:
@bkueng, @tstastny any chanch to get progress in this issue?

tstastny commented:
@bkueng @BazookaJoe1900 -- just remembered I never pushed the corrections.. sorry for the super long delay!! Rebased, and added the requested flag. CI is failing, however. I saw a comment in the log that said something about trailing whitespace.. is this what is keeping it from building? Or something else?

bkueng commented:
> is this what is keeping it from building? Or something else?

Yes, spaces should be removed on empty lines.

tstastny commented:
@bkueng alright all passing. Let me know if there is anything else you would like to see before a merge.
